baremetal-prep/baremetal-prep.sh

```diff
@@ -148,14 +158,26 @@ setup_bridges(){
 
 install_depends(){
   echo "Installing required dependencies..."
-  sudo yum -y install ansible git usbredir golang libXv virt-install libvirt libvirt-devel libselinux-utils qemu-kvm mkisofs 
+  sudo yum -y install ansible git usbredir golang libXv virt-install libvirt libvirt-devel libselinux-utils qemu-kvm mkisofs python3-devel jq ipmitool
```

Comment:
Are these really needed?

* usbredir
* golang
* libXv
* python3-devel

Reply:
Except for python3-devel, those were not added as part of this PR, so assumed the were needed - @schmaustech ?

python3-devel was added since it was documented in the install steps which this is attempting to automate - @rlopez

Reply:
Golang can be removed that is an artifact.  The other two will get installed even if not present as they are dependencies.  Since I was using a ISO image I wanted to have those laid down from ISO instead of pulled in by CDN.   They can be removed but on any host I suspect you will find them installed anyways.

$ sudo rpm -e usbredir
error: Failed dependencies:
	libusbredirparser.so.1()(64bit) is needed by (installed) qemu-kvm-core-15:2.12.0-88.module+el8.1.0+5013+4f99814c.1.x86_64
	usbredir >= 0.7.1 is needed by (installed) qemu-kvm-core-15:2.12.0-88.module+el8.1.0+5013+4f99814c.1.x86_64

$ sudo rpm -e libXv
error: Failed dependencies:
	libXv.so.1()(64bit) is needed by (installed) gstreamer1-plugins-base-1.14.0-4.el8.x86_64
$ sudo rpm -e gstreamer1-plugins-base
error: Failed dependencies:
	libgstapp-1.0.so.0()(64bit) is needed by (installed) spice-server-0.14.2-1.el8.x86_64
	libgstvideo-1.0.so.0()(64bit) is needed by (installed) spice-server-0.14.2-1.el8.x86_64
$ sudo rpm -e spice-server
error: Failed dependencies:
	libspice-server.so.1()(64bit) is needed by (installed) qemu-kvm-core-15:2.12.0-88.module+el8.1.0+5013+4f99814c.1.x86_64
	libspice-server.so.1(SPICE_SERVER_0.10.0)(64bit) is needed by (installed) qemu-kvm-core-15:2.12.0-88.module+el8.1.0+5013+4f99814c.1.x86_64
	libspice-server.so.1(SPICE_SERVER_0.10.1)(64bit) is needed by (installed) qemu-kvm-core-15:2.12.0-88.module+el8.1.0+5013+4f99814c.1.x86_64
	libspice-server.so.1(SPICE_SERVER_0.10.2)(64bit) is needed by (installed) qemu-kvm-core-15:2.12.0-88.module+el8.1.0+5013+4f99814c.1.x86_64
	libspice-server.so.1(SPICE_SERVER_0.10.3)(64bit) is needed by (installed) qemu-kvm-core-15:2.12.0-88.module+el8.1.0+5013+4f99814c.1.x86_64
	libspice-server.so.1(SPICE_SERVER_0.10.4)(64bit) is needed by (installed) qemu-kvm-core-15:2.12.0-88.module+el8.1.0+5013+4f99814c.1.x86_64
	libspice-server.so.1(SPICE_SERVER_0.11.2)(64bit) is needed by (installed) qemu-kvm-core-15:2.12.0-88.module+el8.1.0+5013+4f99814c.1.x86_64
	libspice-server.so.1(SPICE_SERVER_0.12.2)(64bit) is needed by (installed) qemu-kvm-core-15:2.12.0-88.module+el8.1.0+5013+4f99814c.1.x86_64
	libspice-server.so.1(SPICE_SERVER_0.12.3)(64bit) is needed by (installed) qemu-kvm-core-15:2.12.0-88.module+el8.1.0+5013+4f99814c.1.x86_64
	libspice-server.so.1(SPICE_SERVER_0.12.4)(64bit) is needed by (installed) qemu-kvm-core-15:2.12.0-88.module+el8.1.0+5013+4f99814c.1.x86_64
	libspice-server.so.1(SPICE_SERVER_0.12.5)(64bit) is needed by (installed) qemu-kvm-core-15:2.12.0-88.module+el8.1.0+5013+4f99814c.1.x86_64
	libspice-server.so.1(SPICE_SERVER_0.13.1)(64bit) is needed by (installed) qemu-kvm-core-15:2.12.0-88.module+el8.1.0+5013+4f99814c.1.x86_64
	libspice-server.so.1(SPICE_SERVER_0.14.2)(64bit) is needed by (installed) qemu-kvm-core-15:2.12.0-88.module+el8.1.0+5013+4f99814c.1.x86_64
	libspice-server.so.1(SPICE_SERVER_0.6.0)(64bit) is needed by (installed) qemu-kvm-core-15:2.12.0-88.module+el8.1.0+5013+4f99814c.1.x86_64
	libspice-server.so.1(SPICE_SERVER_0.8.1)(64bit) is needed by (installed) qemu-kvm-core-15:2.12.0-88.module+el8.1.0+5013+4f99814c.1.x86_64
	libspice-server.so.1(SPICE_SERVER_0.8.2)(64bit) is needed by (installed) qemu-kvm-core-15:2.12.0-88.module+el8.1.0+5013+4f99814c.1.x86_64
	libspice-server.so.1(SPICE_SERVER_0.8.3)(64bit) is needed by (installed) qemu-kvm-core-15:2.12.0-88.module+el8.1.0+5013+4f99814c.1.x86_64